astropy/stats/sigma_clipping.py

```diff
@@ -385,12 +395,12 @@ def _sigmaclip_fast(self, data, axis=None,
     def _sigmaclip_noaxis(self, data, masked=True, return_bounds=False,
                           copy=True):
         """
-        Sigma clip the data when ``axis`` is None and ``grow`` is not >0.
+        Sigma clip the data when ``axis`` is None and ``grow`` is not
```

Comment:
Total nitpick, but my tendency would be to try to keep a single line and instead delete "the data" or so

Reply:
fixed